fendermint/app/src/cmd/run.rs

```diff
@@ -36,11 +43,44 @@ async fn run(settings: Settings) -> anyhow::Result<()> {
     let ns = Namespaces::default();
     let db = open_db(&settings, &ns).context("error opening DB")?;
 
+    // Blockstore for actors.
     let state_store =
         NamespaceBlockstore::new(db.clone(), ns.state_store).context("error creating state DB")?;
 
     let resolve_pool = CheckpointPool::new();
 
+    if settings.resolver_enabled() {
+        let service =
+            make_resolver_service(&settings, db.clone(), state_store.clone(), ns.bit_store)?;
+
+        let client = service.client();
+
+        let own_subnet_id = settings.resolver.subnet_id.clone();
```

Comment:
I agree with your TODO from above, it'll be great if we can reconcile all the `subnet_id`  config by picking up the value from an IPC setting or, ideally, from genesis.